src/patterns/create-accounts/index.md

```diff
@@ -19,11 +19,11 @@ Help users create an account for your service.
   name: "Create accounts",
   criteria: [
     {
-      text: "reuse information they've already entered",
+      text: "make sure users can reuse information they've already entered",
       anchor: "wcag-reuse-account-information"
     },
     {
-      text: "avoid doing a cognitive test, such as a CAPTCHA, to create an account",
+      text: "make sure that cognitives tests, such as CAPTCHA, are not needed for users to create an account",
```

Comment:
Did you remove the 'a' intentionally? If yes, the 'CAPTCHA' should end in an 's'.

Reply:
```suggestion
      text: "make sure that cognitives tests, such as CAPTCHAs, are not needed for users to create an account",
```